matchms/exporting/__init__.py

```diff
@@ -1,4 +1,8 @@
+from .save_as_json import save_as_json
 from .save_as_mgf import save_as_mgf
 
 
-__all__ = ["save_as_mgf"]
+__all__ = [
+    "save_as_json",
+    "save_as_mgf"
```

Comment:
Please add , so later save_as_* methods only have to change their own line.
```suggestion
    "save_as_mgf",
```